### ISSUE TITLE
Repo

### ISSUE THREAD

**Issue by MetaStazz** [oleksis/youtube-dl-gui#55]
When this forks will make it to official linux (opensuse) repo instead of the old 0.4 version from "MrS0m30n3" ?!

**Comment by oleksis** [comment]
You are right, it is time to create an RPM-based system version.  It would be helpful to create a specification for openSUSE.  You can check this comment and the [packaging for Mageia](https://github.com/oleksis/youtube-dl-gui/issues/43#issuecomment-927142476)

**Comment by oleksis** [comment]
@MetaStazz can try and test the [home repository for openSUSE Tumbleweed](https://build.opensuse.org/package/show/home:oleksis.fraga/yt-dlg) and let us know is work?

```bash
sudo zypper ar -cfp 90 https://download.opensuse.org/repositories/home:oleksis.fraga/openSUSE_Tumbleweed/home:oleksis.fraga.repo
sudo zypper install python38-yt-dlg
yt-dlg
```
